modules/solid_mechanics/src/materials/SymmIsotropicElasticityTensor.C

```diff
@@ -68,6 +68,15 @@ SymmIsotropicElasticityTensor::shearModulus() const
 }
 
 Real
+SymmIsotropicElasticityTensor::youngsModulus() const
+{
+  if (!_E_set)
+    mooseError("Youngs modulus not set");
+
+  return _E;
+}
```

Comment:
I have made a power law hardening material model which uses the young modulus accessor function and I have included a test for it. Thanks!